webrtc.html

```diff
@@ -2075,6 +1826,314 @@
+
+        <p>The <code>RTCRtpSender</code> interface allows an application to control how a given
+        <code>MediaStreamTrack</code> is encoded and transmitted to a remote peer.
+        When attributes on an <code>RTCRtpSender</code> are modified, the encoding is either
+        changed appropriately, or a negotiation is triggered to signal the new encoding
+        parameters to the other side.</p>
+
+        <dl class="idl" title=
+          "interface RTCRtpSender">
+        <dt>readonly attribute MediaStreamTrack track</dt>
+
+        <dd>
+          <p>The <dfn id=
+          "dom-rtpsender-track"><code>RTCRtpSender.track</code></dfn>
+          attribute is the track that is associated with this
+          <code><a>RTCRtpSender</a></code> object.</p>
+        </dd>
+      </section>
+
+      <section>
+        <h3>RTCRtpReceiver Interface</h3>
+
+        <p>The <code>RTCRtpReceiver</code> interface allows an application to control the receipt
+        of a <code>MediaStreamTrack</code>. When attributes on an <code>RTCRtpReceiver</code> are modified, a negotiation is triggered to signal the changes regarding what the application
+        wants to receive to the other side.</p>
+
+        <dl class="idl" title=
+          "interface RTCRtpReceiver">
+        <dt>readonly attribute MediaStreamTrack track</dt>
+
+        <dd>
+          <p>The <dfn id=
+          "dom-rtpreceiver-track"><code>RTCRtpReceiver.track</code></dfn>
+          attribute is the track that is immutably associated with this
+          <code><a>RTCRtpReceiver</a></code> object.</p>
+        </dd>
+      </section>
+
+        <section>
+      <h3>RTCTrackEvent</h3>
+
+      <p>The <code><a href="#event-track">ontrack</a></code>
+      event uses the
+      <code><a>RTCTrackEvent</a></code> interface.</p>
+
+      <p><dfn id="fire-track-event" title="fire track event">Firing an
+      RTCTrackEvent event named <var>e</var></dfn> with an
+      <code><a>MediaStreamTrack</a></code> <var>track</var> means that an event
+      with the name <var>e</var>, which does not bubble (except where otherwise
+      stated) and is not cancelable (except where otherwise stated), and which
+      uses the <code><a>RTCTrackEvent</a></code> interface with the
+      <code><a href="#dom-trackevent-track">track</a></code> attribute
+      set to <var title="">track</var>, MUST be created and dispatched at the
+      given target.</p>
+
+      <dl class="idl" data-merge="RTCTrackEventInit" title=
+      "interface RTCTrackEvent : Event">
+        <dt>Constructor(DOMString type, RTCTrackEventInit
+        eventInitDict)</dt>
+
+        <dt>readonly attribute RTCRtpReceiver receiver</dt>
+
+        <dd>
+          <p>The <dfn id=
+          "dom-trackevent-receiver"><code>receiver</code></dfn> attribute
+          represents the <code><a>RTCRtpReceiver</a></code> object associated with
+          the event.</p>
+        </dd>
+
+        <dt>readonly attribute MediaStreamTrack track</dt>
```

Comment:
I'm worried that when the event was created by media before the SDP, one will not know the stream.